src/tests/functional/plugin/gpu/shared_tests_instances/multi/gpu_remote_blob_tests.cpp

```diff
@@ -8,6 +8,9 @@
 #include "multi/multi_remote_blob_tests.hpp"
 #include "multi/multi_remote_blob_multidevice_test.hpp"
 #include "common_test_utils/test_constants.hpp"
+#include <openvino/runtime/auto/properties.hpp>
+
+using MultiDevice_RemoteBlobAndBindTest = MultiDevice_Test;
```

Comment:
let's use MultiDevice_Test only, bind can be tested through instantiating params with config DEVICE_BIND_BUFFER(YES)